src/deployment/android.md

```diff
@@ -296,33 +296,40 @@ are correct, especially the following:
 
 ## Reviewing the build configuration
 
-Review the default [Gradle build file][gradlebuild],
-`build.gradle`, located in `[project]/android/app` and
-verify the values are correct, especially the following
+Review the default [Gradle build file][gradlebuild] (`build.gradle`) located in 
+`[project]/android/app` and the `local.properties` file located in 
+`[project]/android` to verify the values are correct, especially the following
 values in the `defaultConfig` block:
 
+#### In `build.gradle` file
+
 `applicationId`
 : Specify the final, unique (Application Id)[appid]
 
-`versionCode` & `versionName`
-: Specify the internal app version number,
-  and the version number display string. You can do this by setting
-  the `version` property in the pubspec.yaml file. Consult the version
-  information guidance in the [versions documentation][versions].
-
-`minSdkVersion`, `compilesdkVersion`, & `targetSdkVersion`
-: Specify the minimum API level,
-  the API level on which the app was compiled,
-  and the maximum API level on which the app is designed to run.
-  Consult the API level section in the [versions documentation][versions]
-  for details.
+`compileSdkVersion`
+: Specify the API level Gradle should use to compile your app. For more 
+  information, see the module-level build section in the 
+  [Gradle build file][gradlebuild]. 
 
 `buildToolsVersion`
```

Comment:
This sounds a bit like it's required unless AGP is used.  AGP is almost always used. could it say something like:

> If you're using Android plugin for Gradle 3.0.0 or higher, your project automatically uses a default version of the build tools that the plugin specifies.
> 
> Alternatively, you can specify a version of the build tools.
>

Reply:
Hi @blasten, thank you for pointing this out. In https://github.com/flutter/website/pull/6761/commits/7f8c0760708766c1f7fa2d45dbdcac68e60dd081, I changed the whole `buildToolsVersion` section to the text mentioned. I also thought about adding it as a `{{site.alert.note}}` instead.